src/libsyntax/ext/build.rs

```diff
@@ -135,7 +135,7 @@ pub trait AstBuilder {
 
     fn expr_usize(&self, span: Span, i: usize) -> P<ast::Expr>;
     fn expr_int(&self, sp: Span, i: isize) -> P<ast::Expr>;
-    fn expr_u8(&self, sp: Span, u: u8) -> P<ast::Expr>;
+    fn expr_u32(&self, sp: Span, u: u32) -> P<ast::Expr>;
```

Comment:
These functions are mostly used by syntax extensions, so someone outside the compiler might be using it